src/utils.jl

```diff
@@ -17,7 +17,8 @@ end
 
 function _round_phase(p::Parameter)
     @match p begin
-        PiUnit(_...) => rem(rem(p, 2) + 2, 2)
+        PiUnit(pu, pt) && if pt <: Number
```

Comment:
We should also consider the case where `pu` is a `Symbol` but `pt` is a `Float64` (typed symbolic expression).

Reply:
I changed it here. The issue is left open in case we run into trouble elsewhere. I will mark this as resolved for now.